### DISCUSSION TITLE
Is it possible to upload attachment via URL or Base64 ?

### DISCUSSION THREAD

**Discussion by ticmaflow** [nocodb/nocodb#8462] [Q&A]
Hello,

I need to upload an attachment via URL or Base64 but I do not see anything mentionning this possibility in the documentation.

Going through the other discussions there is mention of an uploadByUrl in #3922 but it seems it doesn't work anymore.

Am I missing something ?

Best regards

**Comment by ticmaflow** [comment]
Oh forgot to mention I also tried upload-by-url and adding an url in the body but I get this error :

{
    "msg": "No attachment provided!"
}

![image](https://github.com/nocodb/nocodb/assets/138558571/1f667f71-6f2a-47cc-8c5b-a0e6a1a57fa8)

**Comment by Pegazse** [comment]
Hi! Did you find a solution to upload by url?

**Comment by ticmaflow** [comment]
Hello! nop sorry following this issue I had to drop nocoDB :(

**Comment by Pegazse** [comment]
Hi! i found a i solution that works for me in n8n. You need to put the image in an array like this [{"url": "https://fr.mypet.com/wp-content/uploads/sites/10/2024/03/Chat-Persan.png"}]

**Comment by ticmaflow** [comment]
Thanks Pegazse!

Valère TARAVELLA
M: +262 (6) 93 04 77 49 // +33 (6) 21 12 68 06

45 rue Alexis de Villeneuve, 97400 Saint Denis

**Comment by shaojjjin** [comment]
I encountered some similar problems. My machine is deployed on the internal network.

Then such an error will occur. How should I solve this problem

"body": "{\"msg\":\"DNS lookup 192.168.5.254(family:undefined, host:192.168.5.254) is not allowed. Because, It is private IP address.\"}",
